### PR TITLE
#1173: improve navigation through guides

### PR DESCRIPTION
**Pull request by romanshoryn**
Hi everyone!

I improved navigation through guides as it was required by the issue #1173.

screen shot 2017-07-08 at 9 48 51 pm

### CODE REVIEW

**Comment by fhemberger** [comment]
PR itself looks good to me, thanks!
Can the localization teams please check the translations of "previous"/"next"?

**Review by fhemberger — APPROVED** [review]

**Comment by vsemozhetbyt** [comment]
`uk` is OK)

**Review by hiroppy — APPROVED** [review]

File: locale/de/site.json [hunk 1]
```diff
@@ -10,6 +10,8 @@
     "by": "von",
     "all-downloads": "Alle Download-Optionen",
     "nightly": "Nightly Builds",
+    "previous": "Bisherige",
```
Comment (fhemberger) [inline locale/de/site.json]: I'd use "zurück" / "weiter" (back/forth instead)

File: locale/it/site.json [hunk 2]
```diff
@@ -10,6 +10,8 @@
     "by": "da",
     "all-downloads": "Tutti i download",
     "nightly": "Build notturne",
+    "previous": "Precedente",
```
Comment (lpinca) [inline locale/it/site.json:13]: It's probably to better to use "Indietro/Avanti".
If the context is "page" then it's ok but "next" should be "Successiva".

**Comment by romanshoryn** [comment]
@fhemberger and @lpinca thanks for the review. I've fixed all issues.

**Comment by fhemberger** [comment]
@romanshoryn Great, thank you!

**Pull request merged**